containerz/containerz.proto

```diff
@@ -208,13 +221,25 @@ message ListResponse {
   Status status = 4;
 }
 
-message StartRequest {
+// A volume can be passed to a container such that it can persist data.
```

Comment:
Should we have more definition here around what a volume is expected to do? I think we want to say that it's backed by persistent storage and must exist across device reboots right?

Reply:
done - I added this to the CreateVolume RPC since this message is only intended to attach an existing volume to a container.